docs/contributing/index.md

```diff
@@ -53,35 +79,167 @@ Tap, click, or hover over the GitHub Octocat icon for options.
+1.  Navigate with the `docs` directory to find the source file to edit.
+1.  Click the {guilabel}`pencil icon` to edit the file in the browser.
 
-Contributors to the Plone Documentation may perform one or many roles.
+    ```{image} /_static/edit-pencil-icon.png
+    :alt: GitHub Edit this file
+    ```
+1.  Make edits, add a commit message, select {guilabel}`Create a **new branch** for this commit and start a pull request`, then click {guilabel}`Propose changes`.
+1.  Make your pull request against the branch `6-dev`.
+1.  Members who subscribe to the repository will receive a notification and review your request.
 
-- **Plone users and developers** use this documentation because it is accurate and actively maintained.
-  People in these roles typically contribute minor corrections.
-  They should read {doc}`setup-build` and {doc}`writing-docs-guide`.
-- **Authors** create Plone Documentation.
-  They should read {doc}`setup-build` and {doc}`writing-docs-guide`.
-  They should also read {doc}`authors` for guidance and tips for writing good technical documentation.
 
+(contributing-large-edits-label)=
 
-(contributing-quality-requirements-label)=
+### Large edits
 
-## Documentation quality requirements
+For large edits, first follow the instructions in {doc}`setup-build`.
+
+Once you have your environment set up, then you can follow the standard practice for making a pull request.
+This practice differs depending on whether you are making contributions to only the core `documentation` files or `volto` files as well.
+
+
+(contributing-documentation-only-label)=
+
+### Working with only the `documentation` repository
+
+This section describes how to make contributions to files in the `documentation` repository only, and excludes `volto` files.
+
+1.  From the project root directory, synch your local `6-dev` branch with its remote.
+    You might need to resolve conflicts.
+
+    ```shell
+    git checkout 6-dev
+    git pull
+    ```
+
+1.  Create a new branch from `6-dev`.
+
+    ```shell
+    git checkout -b <new_branch>
+    ```
+
+1.  Edit files, save, preview, and test.
+    You must run and pass the builds `html` and `linkcheck` without causing new errors.
+
+    ```shell
+    # Optionally clean the builds to avoid cache issues
+    make clean
+    make html
+    make linkcheck
+    ```
+
+    ```{note}
+    Currently there are some errors on the `html` build, mostly due to empty `meta` HTML tags.
+    You are welcome to fix as many errors as you like.
+    You are only responsible to fix errors that you create.
+    ```
+
+    ```{note}
+    Eventually the `spellcheck` build will be required, but not at this time.
+    We welcome improvements to the dictionary.
+    ```
+
+    ```{seealso}
+    {ref}`setup-build-available-documentation-builds-label`.
+    ```
+
+1.  After the builds pass, commit changes to your branch, and push it to the remote repository on GitHub.
```

Comment:
```suggestion
1.  After the build pass, commit changes to your branch, and push it to the remote repository on GitHub.
```

Reply:
There are multiple builds that must pass, not just one.